Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM docker.io/rocker/r-ver:4.3.1
+
+RUN CRAN_LIKE_URL="https://packagemanager.posit.co/cran/__linux__/jammy/2024-04-05"; \
+    printf "options(\n \
+    repos = c(CRAN = '%s'),\n \
+    pak.no_extra_messages = TRUE,\n \
+    pkg.sysreqs = FALSE,\n \
+    pkg.sysreqs_db_update = FALSE,\n \
+    pkg.sysreqs_update = FALSE\n \
+    )\n" \
+    "$CRAN_LIKE_URL" \
+    > "${R_HOME}/etc/Rprofile.site" \
+    && Rscript -e "install.packages('pak', repos = sprintf('https://r-lib.github.io/p/pak/stable/%s/%s/%s', .Platform[['pkgType']], R.Version()[['os']], R.Version()[['arch']]))"
```

Comment:
I don't really know what all of this does/ if it is necessary, I just copied it in from `workflow.pacta`

Reply:
simplified